hvpy/api_groups/jpeg2000/getJp2Image.py

```diff
@@ -0,0 +1,20 @@
+# Contains I/O parameters for getJp2Image
+from datetime import datetime
+from typing import Optional
+from pydantic import BaseModel
+
+
+class getJP2ImageInputParameters(BaseModel):
```

Comment:
This should inherit from HvpyParameters Then you don't need to define the dict function here.

```suggestion
class getJP2ImageInputParameters(HvpyParameters):
```